packages/e2e/cypress.config.ts

```diff
@@ -24,5 +24,5 @@ export default defineConfig({
       return config;
     },
   },
-  videoUploadOnPasses: false,
+  video: false,
```

Comment:
Turned off video recording for cypress, it is CPU intensive and time consuming, and not needed, screenshot should suffice.